lib/tfpdf/wp-tfpdf.php

```diff
@@ -0,0 +1,134 @@
+<?php
+/**
+ * To avoid modifying tFPDF directly we introduce WP_tFPDF, 
+ * A thin wrapper around the parent class to utilize WP_Filesystem 
+ * 
+ * Parent's coding standards are preserved deliberately.
+ */
+// phpcs:disable WordPress.WP.AlternativeFunctions.file_system_read_fwrite, WordPress.WP.AlternativeFunctions.file_system_read_fread, WordPress.WP.AlternativeFunctions.file_system_read_fopen, WordPress.WP.AlternativeFunctions.file_system_read_fclose, WordPress.VIP.FileSystemWritesDisallow
```

Comment:
Why do we need to ignore all of these linters for this file? Shouldn't we be ok here since we're using `$wp_filesystem`?

Reply:
We do need them because `fread`, `fopen`, and `fclose` are present in `Image()`.